packages/mux-player/src/index.ts

```diff
@@ -328,28 +309,48 @@ class MuxPlayerElement extends VideoApiElement {
     newValue: string
   ) {
     super.attributeChangedCallback(attrName, oldValue, newValue);
+    this.#render();
+  }
 
-    // Prevent forwarding the playback-id, the player sets the src attribute.
-    if (
-      attrName != MuxVideoAttributes.PLAYBACK_ID &&
-      MuxVideoAttributeNames.includes(attrName)
-    ) {
-      if (newValue === null) {
-        this.video?.removeAttribute(attrName);
-      } else {
-        this.video?.setAttribute(attrName, newValue);
-      }
-    }
```

Comment:
^^ this is pretty cool, `this.#render();` will now take care of setting and removing the attributes instead of the API calls above here.

Reply:
this is because of template-parts and html.ts?

Reply:
yup :)